bionic/protocols.py

```diff
@@ -314,6 +317,38 @@ def write(self, array, path):
             np.save(file_, array)
 
 
+class DaskProtocol(BaseProtocol):
+    """
+    Decorator indicating that an entity's values always have the
+    ``dask.dataframe.DataFrame`` type.
+
+    These values will be serialized to .dask.pq files.
+    """
+
+    def get_fixed_file_extension(self):
+        return 'dask.pq'
+
+    def validate(self, value):
+        assert isinstance(value, dd.DataFrame)
+
+    def read(self, path, extension):
+        if sys.version_info.major == 2:
+            path = str(path)
+        with warnings.catch_warnings():
+            warnings.filterwarnings('error')
```

Comment:
If we're only concerned about one particular type of warning, how about filtering for that one directly?  Something like `warnings.filterwarnings('error', message=r".*cannot\s+autodetect\s+index.*")
`?

Reply:
Much cleaner, adding!